azurerm/internal/services/cognitive/resource_arm_cognitive_account.go

```diff
@@ -84,10 +84,23 @@ func resourceArmCognitiveAccount() *schema.Resource {
 				}, false),
 			},
 
+			"sku_name": {
+				Type:          schema.TypeString,
+				Optional:      true, // required in 2.0
+				Computed:      true, // remove in 2.0
+				ConflictsWith: []string{"sku"},
+				ValidateFunc: validation.StringInSlice([]string{
+					"F0", "F1", "S0", "S1", "S2", "S3", "S4", "S5", "S6", "P0", "P1", "P2",
```

Comment:
Should these be separated with an underscore to be consistent with the other `sku_name`s?
```suggestion
					"F_0", "F_1", "S_0", "S_1", "S_2", "S_3", "S_4", "S_5", "S_6", "P_0", "P_1", "P_2",
```

Reply:
I don't want to make confusing alternatives, its up to azure to change it